sdk/ai/ai-inference-rest/test/public/chatCompletions.spec.ts

```diff
@@ -25,6 +30,76 @@ describe("chat test suite", () => {
     assert.isNotNull(client.pipeline);
   });
 
+  it("chat regression test", async function () {
+    const headers = { "extra-parameters": "allow" };
+    const body = {
+      messages: [
+        { role: "user", content: "How many feet are in a mile?" },
```

Comment:
Consider adding chat history here (previous system message, previous user message and previous assistant message).
Consider include a user message with image url

Reply:
Thanks!